src/quick-start.md

```diff
@@ -2,45 +2,59 @@
-  are defined inside of Kubernetes
-* An arbitrary number of `PolicyServer` resources, which represent a Deployment of a Kubewarden `policy-server`.
-  This component loads all the policies defined by the administrators and evaluates them
-* A Deployment of `kubewarden-controller`: this is the controller
-  that monitors the `ClusterAdmissionPolicy` resources and interacts
-  with the Kubewarden `policy-server`
+* An arbitrary number of `ClusterAdmissionPolicy` resources: this is how policies are defined inside Kubernetes
+* An arbitrary number of `PolicyServer` resources: this component represents a Deployment of a Kubewarden `policy-server`. The policies defined by the administrators are loaded and evaluated by the Kubewarden `policy-server`
+* A Deployment of `kubewarden-controller`: this is the controller that monitors the `ClusterAdmissionPolicy` resources and interacts with the Kubewarden `policy-server` components
+
+## Installation
+
+> **PREREQUISITES:**
+>
+> Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) *before* installing the kubewarden-controller chart.
+>
+> You can install the latest version of `cert-manager` by running the following commands: 
+>
+```shell
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
 
-## Install
+kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+```
 
-The Kubewarden stack can be deployed using a helm chart.
-Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) and then install the kubewarden-controller chart. For example:
+The Kubewarden stack can be deployed using a `helm` charts as follow:
 
-```console
+```shell
 helm repo add kubewarden https://charts.kubewarden.io
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
-kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+
 helm install --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
+
 helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
 ```
 
-This will install on the Kubernetes cluster:
-* `kubewarden-crds`  will register the
-`ClusterAdmissionPolicy` and `PolicyServer` Custom Resources.
+The following components should be installed inside the `kubewarden` namespace in your Kubernetes cluster:
+
+* `kubewarden-crds` which will register the
+`ClusterAdmissionPolicy` and `PolicyServer` Custom Resources
 
 * `kubewarden-controller` with
-the default configuration. It will create a `PolicyServer` resource named `default`. The components of the
-Kubewarden stack will be deployed inside of a Kubernetes Namespace
-called `kubewarden`.
+a default configuration and which will create a `PolicyServer` resource named `default`. 
```

Comment:
Nit: 
Nit:

```suggestion
a default configuration, and which will create a `PolicyServer` resource named `default`. 
```
(Oxford comma; who creates the PolicyServer resource, the default configuration, or kubewarden-controller?)